cpp/misra/src/qlpack.yml

```diff
@@ -1,3 +1,5 @@
-name: misra-cpp-coding-standards
+name: codeql/misra-cpp-coding-standards
 version: 2.13.0-dev
-libraryPathDependencies: common-cpp-coding-standards
+dependencies:
```

Comment:
We should add the `defaultSuiteFile`, `suites`, `license`, and `description` properties.

The `defaultSuiteFile` helps with running the default selected suite when our package is provided to `codeql database analyze` and the `suites` informs the CodeQL cli of the possible suites and turns them into well-known suites so you can run them without providing the full path.

Reply:
Good point. I think running just the MISRA rules would make sense.

I believe we can archive that with an inline suite definition, something like

```yml
defaultSuite:
 qlpack: codeql/autosar-cpp-coding-standards
 include:
   tags contains: /^external/autosar/id/m.*/
```